process/phases.md

```diff
@@ -137,7 +138,8 @@ overseen by the Community Group.
 
 #### What about licenses?
 
-The spec and spec forks will move to a
+The [W3C Community Contributor License Agreement (CLA)](https://www.w3.org/community/about/process/cla/) governs contributions to the Community Group spec.
```

Comment:
```suggestion
The [W3C Community Contributor License Agreement (CLA)](https://www.w3.org/community/about/process/cla/) governs contributions to the spec and proposal repositories.
```

Reply:
Suggested different wording because there is only one spec repo, and we want to make sure to cover the proposal repos, too.